docs/docs/additional_resources/tutorials.mdx

```diff
@@ -1,155 +1,55 @@
 # Tutorials
 
-Below are links to tutorials and courses on LangChain. For written guides on common use cases for LangChain, check out the [use cases guides](/docs/use_cases).
+### Different materials
```

Comment:
`Different Materials` isn't super descriptive as a name for me. Should probably be `##`

Reply:
@eyurtsev  I'd like also to see something more descriptive. Do you have something on your mind? I need several options.